app/assets/stylesheets/utilities/_overrides.scss

```diff
@@ -0,0 +1,39 @@
+@import "govuk_publishing_components/all_components";
```

Comment:
Why does this need to be imported in so many places?

Reply:
@alex-ju - I have an issue where if I remove this line, it cannot find the helpers or variables. Have you come across this issue?

Reply:
Fixed on this commit:

https://github.com/alphagov/collections-publisher/pull/674/commits/8c9df2018ac4be803495fa73926bf137804138e2